components/CustomTable.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+} from '@mui/material';
+
+const CustomTable = ({ headerRow, data }) => {
+  return (
+    <TableContainer>
+      <Table stickyHeader aria-label='sticky table'>
+        <TableHead>
+          <TableRow>
+            {headerRow?.map((heading) => (
+              <TableCell
+                key={heading.key}
+                style={{ minWidth: '150px', fontWeight: 'bold' }}
```

Comment:
avoid the style prop, do the styling in the scss

Reply:
Which app structure do you prefer? (_Note the positioning of the CustomTable.js and CustomTable.module.scss_)
1) Keeping the scss/css files of the components and the pages inside the style folder:

RATE_CARD_APP
>.next
>components
  >**CustomTable.js**
  >SelectMenu.js
>styles
  >Home.module.scss
  >**CustomTable.module.scss**

2) Keeping the scss/css of pages inside the style folder and for the components inside the component folder:

RATE_CARD_APP
>.next
>components
  >customTable
    >**CustomTable.js**
    >**CustomTable.module.scss**
  >SelectMenu.js
>styles
  >Home.module.scss